src/components/task/breadcrumbs.cjsx

```diff
@@ -1,72 +1,30 @@
 React = require 'react'
-BS = require 'react-bootstrap'
 {TaskStore} = require '../../flux/task'
-{StepPanel} = require '../../helpers/policies'
 
 _ = require 'underscore'
 
 CrumbMixin = require './crumb-mixin'
+Breadcrumb = require './breadcrumb'
+BindStoreMixin = require '../bind-store-mixin'
 
 module.exports = React.createClass
   displayName: 'Breadcrumbs'
 
-  mixins: [CrumbMixin]
+  mixins: [CrumbMixin, BindStoreMixin]
```

Comment:
Is listening to the store here necessary? I'm worried about to many components all listening to store changes and recursively re-rendering rather than a shared parent component re-rendering once.